### PR TITLE
Add istioctl install, an alias for manifest apply

### PR DESCRIPTION
**Pull request by ostromart**
This provides an alias so that istioctl install == istioctl manifest apply. 
This was a request from UX group weekly meeting - we can deprecate manifest apply and eventually remove it.

### CODE REVIEW

File: operator/cmd/mesh/manifest-apply.go [hunk 1]
```diff
@@ -90,26 +90,62 @@ func manifestApplyCmd(rootArgs *rootArgs, maArgs *manifestApplyArgs) *cobra.Comm
...
 }
 
+// InstallCmd in an alias for manifest apply.
+func InstallCmd() *cobra.Command {
+	rootArgs := &rootArgs{}
+	macArgs := &manifestApplyArgs{}
+
+	mac := &cobra.Command{
+		Use:   "install",
+		Short: "Applies an Istio manifest, installing or reconfiguring Istio on a cluster.",
```
Comment (sdake) [inline operator/cmd/mesh/manifest-apply.go]: For a follow-up PR for @esnible, it may make sense to simplify the wording for short such as "Install Istio".

File: operator/cmd/mesh/manifest-apply.go [hunk 2]
```diff
@@ -90,26 +90,62 @@ func manifestApplyCmd(rootArgs *rootArgs, maArgs *manifestApplyArgs) *cobra.Comm
...
+		Long:  "The install generates an Istio install manifest and applies it to a cluster.",
+		// nolint: lll
+		Example: `  # Apply a default Istio installation
+  istioctl install
+
+  # Enable grafana dashboard
+  istioctl install --set values.grafana.enabled=true
+
+  # Generate the demo profile and don't wait for confirmation
+  istioctl install --set profile=demo --skip-confirmation
```
Comment (sdake) [inline operator/cmd/mesh/manifest-apply.go]: this will be affected by the work @dgn and @howardjohn are doing in extensions and telemetry. See document here: https://docs.google.com/document/d/1jJ4kkALCUPWIlz_Ldu6jpeelMNxyY_SZDMaZmeSpfkM

**Review by sdake — APPROVED** [review]
May need a little refinement prior to 1.6 release.

LGTM for now.

**Review by esnible — APPROVED** [review]
